aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -24,3 +24,13 @@ proposals:
           git_hash: ~
       - DefaultGasWithOverride:
           feature_version: 13
+  - name: step_3_enable_checking_unstable_bytecode
```

Comment:
@rahxephon89 , the content of this proposal (step) looks good. However, this change in `release.yaml` could be removed from this PR. We are working on changing the "release build" process so that each feature will have its own YAML file. Please add this proposal later (probably next week) when the process is ready.

Reply:
Thanks @junkil-park! I will apply this to the new process